java/pom.xml

```diff
@@ -11,8 +11,12 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <arrow.version>3.0.0</arrow.version>
+        <arrow.version>7.0.0-20220223174055-ed836ca-dremio</arrow.version>
```

Comment:
Can we add Dremio's repo to the POM file:
https://maven.apache.org/guides/mini/guide-multiple-repositories.html